config.go

```diff
@@ -46,6 +47,15 @@ func LoadConfig(path string, fallback cfg.Config) (config cfg.Config, err error)
 		config.GPSd.Addr = config.GPSdAddrLegacy
 	}
 
+	// preventing malicious configuration that exposes files from outside the forms folder
+	if strings.Contains(config.FormsPath, "..") {
+		config.FormsPath = "."
+	}
+
+	// clean up FormsPath (normalizes trailing slashes, and embedded '.' )
+	config.FormsPath = path.Clean(config.FormsPath)
+	config.FormsPath = strings.ReplaceAll(config.FormsPath, "\\", "/")
```

Comment:
refactored to use .Replace(..., -1)
I cannot avoid the Replace: config.FormsPath is used for building the URLs as well as building local file paths. On Windows, it would break building URLs.